packages/core/admin/server/services/api-token.js

```diff
@@ -195,7 +195,7 @@ const create = async (attributes) => {
     //   data: attributes.permissions.map(action => ({ action, token: apiToken })),
     // });
     await Promise.all(
-      attributes.permissions.map((action) =>
+      uniq(attributes.permissions).map((action) =>
```

Comment:
I don't think it's necessary to trim. However, we should add a check to ensure we're referencing actions from the content-api action's provider

Reply:
True, if it's not registered it won't matter anyway.

I wrote [this](https://github.com/strapi/strapi/tree/api-token-v2/only-create-registered-permissions) right after we wrote the getActionsMap method, but it checks that rather than directly checking the ability, so I need to rewrite it.

Reply:
Not sure about the link you shared. Where should I look ? :sweat_smile: 

But yeah, what we could do is directly add a custom yup field validation for the permissions

Reply:
Not really related to the issue this PR aims to fix though